docs/tutorial/automated-testing.md

```diff
@@ -116,6 +114,142 @@ driver.wait(() => {
 driver.quit()
 ```
 
+## Using Playwright
+
+[Microsoft Playwright](https://playwright.dev) is an end-to-end testing framework built
+using browser-specific remote debugging protocols, similar to the [Puppeteer] headless
+Node.js API but geared towards end-to-end testing. Playwright has experimental Electron
+support via Electron's support for the [Chrome DevTools Protocol] (CDP).
+
+### Install dependencies
+
+You can install Playwright through your preferred Node.js package manager. The Playwright team
+recommends using the `PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD` environment variable to avoid
+unnecessary browser downloads when testing an Electron app.
+
+```sh npm2yarn
+PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm install --save-dev playwright
+```
+
+Playwright also comes with its own test runner, Playwright Test, which is built for end-to-end
+testing. You can also install it as a dev dependency in your project:
+
+```sh npm2yarn
+npm install --save-dev @playwright/test
+```
+
+:::caution Dependencies
```

Comment:
Small nit: The callout formats here (`:::caution`, `:::info`, etc) don't seem to work with GitHub's rich text format. This might be fine and show up properly in the docs themselves, just wanted to call it out just in case.

Reply:
Yep, this is admonition syntax baked into Docusaurus 2: https://docusaurus.io/docs/next/markdown-features/admonitions

When creating new tutorials, I've been using the website repo to create deploy previews to show how they look on the live website: https://electronjsorg-new-pr-152.herokuapp.com/docs/latest/tutorial/automated-testing/#using-playwright

Reply:
Oh awesome, sorry, just saw you included that in the PR description and I missed it 🙂  Looks great!